specreduce/tracing.py

```diff
@@ -164,9 +162,9 @@ class KosmosTrace(Trace):
         direction is axis 1.
     bins : int, optional
         The number of bins in the dispersion (wavelength) direction
-        into which to divide the image. Use fewer if KosmosTrace is
-        having difficulty, such as with faint targets.
-        Minimum bin size is 4. [default: 20]
+        into which to divide the image. If not set, defaults to one bin
+        per dispersion (wavelength) pixel in the given image. If set,
+        requires a minimum of 4 bins. [default: None]
```

Comment:
is a lower limit of 4 bins sufficient or should we have a limit based on the order of the input model?